README.md

```diff
@@ -48,7 +53,11 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+#if town includes belle reject it.
+belle = town[:residents].find {|person| person.eql?("Belle")}
+
+town[:castle][:residents] + " and " + belle
+
 ```
```

Comment:
You selected "Belle" with that first line, but you need to delete her. You can do that like this...

```rb
belle = town[:residents].delete "Belle"
```

The next step is to add her to the `guests` array, not the `residents` string. Knowing that, how would you approach this?